roles/KubevirtTemplateValidator/defaults/main.yml

```diff
@@ -2,5 +2,3 @@
 # defaults file for KubevirtTemplateValidator
 wait: true
 validator_image: kubevirt-template-validator
-# this comes from the CR, we must diverge from validator_$SOMETHING standard naming
-version: "{{ validator_tag |default('v0.6.2') }}"
```

Comment:
Same here. The CR exposes the version using the `version` variable. And the defaults have to follow the suit. We can replace the default with latest, but that is it.

Reply:
In this case I think we should just omit the version from the CR we ship. This duplication seems harmful. We should instead document/validate the schema of the CR (in another PR ofc). Maybe also for the templates?

Reply:
We could omit the version from the validator CR. There is a good reason to have it for templates though, it allows you to install older versions of templates in case you need the older definition.

Reply:
ack, works for me.